server/ansible/README.org

```diff
@@ -44,11 +48,11 @@ Create an inventory file by copying the example file
 can be created anywhere, but for the purposes of this document we
 assume that it is in ~/tmp/pbench-server.hosts~.
 
-In addition to the hosts, the inventory file should contain a variable
-whose value should be the URL where the server config file can be
-found. We provide a generic example inventory file, but a local
-installation should provide its own example inventory file with
-that variable already set to point to the server config file
+In addition to the hosts, the inventory file should define a
+=configurl= variable whose value should be the URL where the server
+config file can be found. We provide a generic example inventory file,
+but a local installation should provide its own example inventory file
+with that variable already set to point to the server config file
```

Comment:
Simplify by explaining all the changes that need to be made from the generic example file to a working file.